examples/adb-vnet-injection/variables.tf

```diff
@@ -15,7 +15,7 @@ variable "dbfs_prefix" {
 
 variable "node_type" {
   type    = string
-  default = "Standard_DS3_v2"
+  default = "Standard_E8ds_v4"
```

Comment:
why this? it's better to leave it small

Reply:
I am not sure what caused this specific merge conflict as this isn't a part I touched. I agree with you on this.